sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb

```diff
@@ -672,6 +692,89 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.2.1 Individual runs with models from MMDetection (Preview)\n",
+    "\n",
+    "In addition to the model algorithms supported natively bu AutoML, you can launch individual runs to explore any model algorithm from MMDetection that supports object detection. Please refer to this [documentation](https://github.com/open-mmlab/mmdetection/blob/v2.28.2/docs/en/model_zoo.md) for the list of models.\n",
+    "\n",
+    "While you can use any model from MMDetection to support this task, we have curated a set of model algorithms in our registry. We provide a set of sensible default hyperparameters for these model algorithms. You can fetch the list of curated model algorithms using code snippet below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client = MLClient(credential, registry_name=\"azureml-staging\")\n",
+    "\n",
+    "models = ml_client.models.list()\n",
+    "object_detection_models = []\n",
+    "for model in models:\n",
+    "    model = ml_client.models.get(model.name, label=\"latest\")\n",
+    "    if model.tags['task'] == 'image-object-detection':\n",
+    "        object_detection_models.append(model.name)\n",
+    "\n",
+    "object_detection_models"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you wish to try a model algorithm (say `deformable_detr_r50_16x2_50e_coco`), you can specify the job for your AutoML Image runs as follows:"
```

Comment:
is there a reason you chose this model? I haven't seen it before, should we use models that are going to be curated?

Reply:
Yeah, we should use curated models. I reached out to @rjaincc for the same info on OD/IS models. Planning to update this later.

Reply:
Updated all notebooks to use curated models